tests/Integration/Security/Verifier/JwtTokenVerifierTest.php

```diff
@@ -60,4 +63,19 @@ public function testUnsuccessfulVerification(): void
     {
         self::assertFalse($this->subject->isValid($this->createMock(Token::class)));
     }
+
+    private function createToken(): Plain
```

Comment:
lets name it like `createNotValidToken` or something. for the better understanding